tests/test_sto_v2.py

```
@@ -0,0 +1,56 @@
+from pathlib import Path
+from pathlib import Path
```

Ruff (0.14.14)

[error] Redefinition of unused `Path` from line 1: `Path` redefined here

Remove definition: `Path`

(F811)